src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -60,15 +60,19 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
   });
 
   if (__DEV__) {
-    warning(
-      browserSupportsResizeObserver && hasOverflow,
-      'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
-    );
+    if (hasOverflow) {
+      warning(
```

Comment:
I had to go looking for the documentation on `warning`.  That's one counter-intuitive function :(.  This expression is correct, but, at first , I was going to ask that check be `!browserSupportResizeObserver`, but I see the warning is thrown when the expression doesn't evaluate to `true`.